core/server/models/settings.js

```diff
@@ -84,7 +84,9 @@ Settings = ghostBookshelf.Model.extend({
                 return;
             }
 
-            return validation.validateActiveTheme(themeName);
+            return validation.validateActiveTheme(themeName).catch(errors.ActiveThemeNotFoundError, function (error) {
+                return self.save({value: error.fallbackTheme}, {patch: true});
```

Comment:
Not sure if that is the correct way to update the current setting or if I should call Settings.edit or something similar.